agent/profile

```diff
@@ -24,13 +24,7 @@ then
     echo "                Setup has failed - could not determine the proper environment prefix: \"$prefix\"" >&2
     echo "                You will need to fix it before working with pbench." >&2
 else
-    if [[ -f /opt/configtools/bin/getconf.py ]]
-    then
-	pathins /opt/configtools/bin
-	# no need to set PYTHONPATH if configtools is installed
-    else
-	pathins $prefix/configtools/bin
-    fi
+    pathins $prefix/common/bin
```

Comment:
Don't we need an enhanced PYTHONPATH?

Reply:
Maybe not  here - it *is* enhanced in the trampoline code.

Reply:
... except the trampoline code is on the server, so maybe we *do* need it on the agent - I feel dizzy :-)

Reply:
I think you do need PYTHONPATH - if you don't think so, can you explain why not?